src/enumo/scheduler.rs

```diff
@@ -137,7 +137,7 @@ impl Scheduler {
         egraph: &EGraph<L, SynthAnalysis>,
         ruleset: &Ruleset<L>,
     ) -> EGraph<L, SynthAnalysis> {
-        self.run_internal(egraph, ruleset, None, 200_000)
```

Comment:
why did you change this back? @oflatt  lowered it in https://github.com/uwplse/ruler/pull/175

Reply:
I'll put it back, wanted to see what would happen.